docs/adding-new-boards.rst

```diff
@@ -0,0 +1,107 @@
+.. adding-new-boards:
+
+.. highlight:: none
+
+Adding a new board
+======================
+
+First make sure the particular chip and chip family is supported, see
+below.
+
+Adding a new board is relatively easy. You will need:
+
+-  A new target description in .json format. It is probably easiest to
+   copy an existing description from the same chip and adjust it to your
+   needs.
+   TODO: make it possible to inherit these definitions from a base .json
+   file to avoid duplication.
```

Comment:
Can you add a link somewhere here to the build tags documentation? With the build tags docs on a separate page it's not that obvious anymore that they're related. A link can be added with for example ``:ref:`build tags <buildtags>` ``. You can say something like "For more information on the `build-tags` field, see <build tags>" (or however you'd like to formulate that).

Also, maybe this TODO needs to be removed. It was mostly a hint to myself. Such a thing doesn't really belong in the documentation.

Reply:
The PR seems like it is turning into more work then you having updated the pages yourself, which was certainly not my intention. Maybe we should just forget the whole idea, and just do nothing at present?